pkg/devfile/parser/parse.go

```diff
@@ -81,6 +81,8 @@ type ParserArgs struct {
 	Context context.Context
 	// K8sClient is the Kubernetes client instance used for interacting with a cluster
 	K8sClient client.Client
+	// HTTPTimeout overrides the request and response timeout values for reading a parent devfile reference from the registry.  If a negative value is specified, the default timeout will be used.
+	HTTPTimeout *int
```

Comment:
any particular reason we make it a pointer?

Reply:
yes, it's to determine if this value is unset or not.  If unset, we will use the default value of 30s